pyccel/parser/semantic.py

```diff
@@ -2638,8 +2638,9 @@ def _visit_FunctionCall(self, expr):
             cls_variable = self._assign_lhs_variable(expr.get_user_nodes(Assign)[0].lhs, d_var, expr, new_expression, False)
             self._additional_exprs[-1].extend(new_expression)
             args = (FunctionCallArgument(cls_variable), *args)
-            # TODO check compatibility
             # TODO treat parametrized arguments.
```

Comment:
> TODO treat parametrized arguments.

Do you know if this works?

```python
class A:
    def __init__(self, x : int, y : bool):
        self.x = x
        self.y = y

if __name__ == '__main__':
    a = (3, True)
    my_class_var = A(*a)
```

Reply:
It works and I added a test for it in https://github.com/pyccel/pyccel/pull/1576/commits/5d93f602a8f9a77b8420dff5ea3ed42f49bdfac6
I already deleted the TODO comment in https://github.com/pyccel/pyccel/pull/1576/commits/c22d618b3bef773f237fe7c51c99ecc10426f3df

Reply:
Ah cool :)